start.sh

```diff
@@ -2,4 +2,6 @@
 
 /usr/sbin/crond
 python manage.py migrate
+nginx
+echo 'yes' | python manage.py collectstatic
```

Comment:
remove echo

Reply:
I tried but this doesn't work
`yes | python manage.py collectstatic `

Reply:
Though ` yes yes|python manage.py collectstatic` works.
Should I use this instead?

Reply:
What is the error thrown by Django?
It should most probably because while the postgres container is booting up, it tries to connect to it and fails. You can try using scripts like [wait-for-it](https://github.com/vishnubob/wait-for-it/blob/master/wait-for-it.sh) to wait for the postgres container to boot up. This script should go as the entrypoint of the container

Reply:
python manage.py collectstatic needs yes or no confirmation,if we dont enter yes or no, then it doesn't run.
Using `yes` just outputs `y` , while using echo yes, or `yes yes` outputs yes

Reply:
add  --noinput flag

Reply:
ok